labs/lab9/basicapp.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# A simple Flask demo app that demonstrates one basic route, favicons,
+# and static files. Written for my tutorial,
+# Deploying Python Web Applications with nginx and uWSGI Emperor.
+# <https://go.chriswarrick.com/pyweb>
+# Copyright © 2016-2017, Chris Warrick.  All rights reserved.
+# Licensed under the 2-clause BSD license.
+
+from flask import Flask
+app = Flask(__name__)
+
+
+@app.route("/")
+def index():
```

Comment:
I'd make this a slightly more interesting Flask app than just a one-liner with an index route. There should be examples available like those you linked earlier.